packages/flame/test/effects/move_effect_test.dart

```diff
@@ -2,14 +2,19 @@ import 'dart:math';
 
 import 'package:flame/components.dart';
 import 'package:flame/effects.dart';
+import 'package:flame/src/test_helpers/random_test.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 import 'effect_test_utils.dart';
 
-void main() {
-  final random = Random();
+class Elements {
```

Comment:
Is this `Elements` the same as the one in `combined_effect_test`? Wondering if we can somehow prevent duplicate code

Reply:
Did a quick look at it again and looks like we can have some sort of base class with the duplicate code like the `randomVector2`

Reply:
Indeed, there seems to be a fair amount of code duplication there. However, I don't think it's within the scope of this PR to fix that, is it?

Reply:
I think the code duplication for the `Elements` class can be fixed in this PR since it is introduced in this PR, it should be fairly quick to fix I think?

Reply:
It wasn't introduced in this PR -- I merely took all the initialization code at the top of each `main()` and moved it into a class. But you're right about that it should be fairly quick to fix.

Reply:
ok done